.github/workflows/production.yml

```diff
@@ -33,7 +33,7 @@ jobs:
 
       - run: echo '//npm.pkg.github.com/:_authToken=${{secrets.GITHUB_TOKEN}}' >> .npmrc
 
-      - run: npm install
+      - run: npm install --legacy-peer-deps
```

Comment:
could you explain why we need `--legacy..` here? Is it because there are other deps which need to upgraded as well, to actually remove this flag?

Reply:
`react-scripts`/`craco` require Typescript 4, so we have to use `--legacy-peer-deps` to install Typescript 5. Unfortunately there's no version of `react-scripts`/`craco` which use Typescript 5